docs/organization/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -23,3 +25,9 @@ The LaunchDarkly integration is only available to organizations with a Business
 1. Navigate to **Settings > Integrations > LaunchDarkly**
 
 2. Follow the full [LaunchDarkly installation instructions](https://docs.launchdarkly.com/integrations/sentry).
+
+## Change Tracking Integration
```

Comment:
we should mention that there's another step needed to set up feature flags completely (e.g. for eval tracking, they need to set up the platform-specific language for their sdk) & link to a full list of platforms (a product index page, when we create it, would make sense)

Reply:
link to eval tracking setup: `/product/explore/feature-flags/#set-up-your-language-specific-sdk`